pyvista/report.py

```diff
@@ -225,10 +233,11 @@ class Report(scooby.Report):
     def __init__(  # noqa: PLR0917
         self,
         additional=None,
-        ncol=3,
-        text_width=80,
-        sort=False,  # noqa: FBT002
-        gpu=True,  # noqa: FBT002
+        ncol: int = 3,
+        text_width: int = 80,
+        sort: bool = False,  # noqa: FBT001, FBT002
+        gpu: bool = True,  # noqa: FBT001, FBT002
+        downloads: bool = False,  # noqa: FBT001, FBT002
```

Comment:
+1 for default value being `False`, as some users might no want to display private paths when reporting an issue.